lib/ansible/modules/cloud/amazon/ec2_snapshot_copy.py

```diff
@@ -143,7 +143,7 @@ def copy_snapshot(module, ec2):
             )
 
     except WaiterError as we:
-        module.fail_json(msg='An error occurred (%s) waiting for the snapshot to become available. (%s)' % (we.message, we.reason))
+        module.fail_json(msg='An error occurred waiting for the snapshot to become available. (%s)' % str(we), exception=traceback.format_exc())
     except ClientError as ce:
         module.fail_json(msg=ce.message, exception=traceback.format_exc(), **camel_dict_to_snake_dict(ce.response))
```

Comment:
Thanks for fixing this. Do you mind fixing this one as well? ce.message won't exist on python3.
`module.fail_json(msg=str(ce), exception=traceback.format_exc(), **camel_dict_to_snake_dict(ce.response))`